### PR TITLE
Fix (personal desk page refreshed): When personal desk is selected, o…

### PR DESCRIPTION
**Pull request by Aca-jov**
…n refresh the same view is returned

SD-1951

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by plamut** [comment]
:+1:

**Comment by mugurrus** [comment]
Desk test should be updated

File: client/app/scripts/superdesk-desks/desks.js [hunk 1]
```diff
@@ -499,7 +499,7 @@
                                     userDesks = desks;
                                     if (desks._items.length) {
                                         if (!this.activeDeskId || !_.find(desks._items, {_id: this.activeDeskId})) {
```
Comment (ioanpocol) [inline client/app/scripts/superdesk-desks/desks.js]: on fetchCurrentDeskId() delete the === '' condition from if
and here change the code:
                                       f (this.activeDeskId === '') {
                                            this.activeDeskId = 'personal'
                                        } else if (this.activeDeskId === null || !_.find(desks._items, {_id: this.activeDeskId})) {
                                            this.activeDeskId = desks._items[0]._id;
                                        }

File: client/app/scripts/superdesk-desks/desks.js [hunk 2]
```diff
@@ -658,7 +658,9 @@
                                 .then(angular.bind(this, function(desks) {
                                     userDesks = desks;
                                     if (desks._items.length) {
-                                        if (!this.activeDeskId || !_.find(desks._items, {_id: this.activeDeskId})) {
+                                        if (this.activeDeskId === ''){
+                                            this.activeDeskId = '';
```
Comment (mugurrus) [inline client/app/scripts/superdesk-desks/desks.js]: Why assign again the existing value?

**Comment by amagdas** [comment]
Is this still a valid PR?

**Comment by amagdas** [comment]
Ping again @Aca-jov any news on this one?

**Comment by ioanpocol** [comment]
Created a new PR based on it:
https://github.com/superdesk/superdesk/pull/658

**Pull request closed without merging**